cdp/webhook.py

```diff
@@ -165,6 +165,12 @@ def update(
         final_notification_uri = notification_uri or self.notification_uri
         final_event_type_filter = event_type_filter or self.event_type_filter
 
+        # wallet ID is required for wallet activity event type filter, but we do not support updating it just yet, this will be added in the future
+        if self.event_type == WebhookEventType.WALLET_ACTIVITY:
+            final_event_type_filter.actual_instance.wallet_id = (
+                self.event_type_filter.actual_instance.wallet_id
```

Comment:
Can we add a unit test for these in a follow-up?

Reply:
yes sir